components/filters/FilterContainer.tsx

```diff
@@ -0,0 +1,36 @@
+import Stack from '@mui/material/Stack'
+import { SelectChangeEvent } from '@mui/material/Select'
+import { useState } from 'react'
+import FilterDropdown from './FilterDropdown'
+
+const FilterContainer = () => {
+  const [roleSelection, setRoleSelection] = useState(undefined)
+  const [programSelection, setProgramSelection] = useState(undefined)
+
+  const handleRoleChange = (event: SelectChangeEvent) => {
+    setRoleSelection(event.target.value as string)
+  }
+
+  const handleProgramChange = (event: SelectChangeEvent) => {
+    setProgramSelection(event.target.value as string)
+  }
+
+  return (
```

Comment:
The Figma mobile sketch is showing the two FilterDropdowns in a row, but it appears to not display correctly on a mobile phone (chrome dev tools). Do we want to shift it to a column at the xs breakpoint? 
    direction={{xs: 'column', md: 'row'}}

Reply:
I just made that change. Thanks for catching that!